From ca2a87ab7454710725d9089a7276317f7db78811 Mon Sep 17 00:00:00 2001
From: "Antoine Martin" <dev@ayakael.net>
Date: Wed, 15 Dec 2021 22:17:24 +0000
Subject: [PATCH 1/1] Disable package validation
	
Disable package validation which breaks our build.
There's no need to run validation in RPM packages anyway.
(See https://github.com/dotnet/runtime/pull/60881)
---
 eng/packaging.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/packaging.targets b/eng/packaging.targets
index 9fe4b569e5f..ac41fe1eb02 100644
--- a/eng/packaging.targets
+++ b/eng/packaging.targets
@@ -7,7 +7,7 @@
       impact on build reliability. https://github.com/dotnet/runtime/issues/60883 tracks fixing the
       flakiness and removing this condition.
     -->
-    <EnablePackageValidation Condition="'$(DotNetBuildFromSource)' != 'true'">true</EnablePackageValidation>
+    <EnablePackageValidation Condition="'$(DotNetBuildFromSource)' != 'true'">false</EnablePackageValidation>
     <!-- Don't restore prebuilt packages during sourcebuild. -->
     <DisablePackageBaselineValidation Condition="'$(DotNetBuildFromSource)' == 'true'">true</DisablePackageBaselineValidation>
     <PackageValidationBaselineVersion Condition="'$(PackageValidationBaselineVersion)' == ''">$([MSBuild]::Subtract($(MajorVersion), 1)).0.0</PackageValidationBaselineVersion>
-- 
2.34.1

